plugins/Marketplace/lang/en.json

```diff
@@ -153,6 +152,7 @@
         "PluginDownloadLinkMissingPremium": "The %1$s plugin could not be installed. Please check your license. If you have an active subscription for the %1$s plugin, try installing it manually using a zip file downloaded from your %2$sMarketplace account%3$s. %4$sLearn more%5$s.",
         "PluginDownloadLinkMissingFree": "The %1$s plugin could not be installed, try installing it manually using a zip file from this %2$slink%3$s. %4$sLearn more%5$s.",
         "PluginDownloadLinkMissingDescription": "Unable to fetch download link for %1$s plugin, try installing the plugin manually from marketplace. %2$sLearn more%3$s.",
-        "CreatedBy" : "Created by"
+        "CreatedBy" : "Created by",
+        "OverviewPluginSubscriptionsMissingLicenseMessage": "You do not have a license key set. If you have purchased a plugin subscription, go to the %1$sLicense%2$s key page and enter your key. If you are interested in purchasing a plugin, visit the %3$sMarketplace%4$s to explore the available plugins and bundles."
```

Comment:
Should the link to the license key page just be around the word _license_? I'd probably put it around the whole _license key page_ as that is where you will land. Links should be self-descriptive for assistive tech such as screen readers.

Reply:
@michalkleiner Updated as suggested